package.json

```diff
@@ -145,7 +145,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "npm run lint-staged",
-      "commit-msg": "npm run commitlint -- -E HUSKY_GIT_PARAMS"
+      "commit-msg": "npm run commitlint -E HUSKY_GIT_PARAMS"
```

Comment:
Why you change this?

Reply:
Oh sorry- I had some issues with git, and I changed that line while troubleshooting.

Reply:
You must also sign our CLA. Thanks